docs/docker-testnet.md

```diff
@@ -0,0 +1,35 @@
+# Join the testnet with Docker
+
+You must have [docker installed](https://docs.docker.com/engine/installation/) on a host machine.
+
+## Docker Image
+
+Docker image is automatically build and published on [DockerHub](https://hub.docker.com/r/aetrnty/epoch/). All tagged source code versions have their own docker image tag as well. Latest tagged ("stable") version is tagged with `latest`.
+Master branch of the source code is tagged as `dev`.
+
+To pull the latest "stable" image run:
+```bash
+docker pull aetrnty/epoch
+```
+
+## Start a Node
```

Comment:
How to stop the node? (For saving power at end of testing.)

Reply:
Well, I tried to abstain from documenting docker itself instead of our image but here you go: 161a61b